Dream-Home-PEABOT/FEE_DreamHome
# Report display

### What does this PR do?
- Display final report
- Add missing styling
- Installs `moment.js`
### Where should the reviewer start?
- `src/components/Report/Report.tsx`
- `src/components/ReportCategory/ReportCategory.tsx`

### How should this be manually tested?
- run `npm install` to install missing dependencies
- navigate to the `generate_report` route with `http://localhost:3001/generate_report`
- make sure to run `npm install -g json-server`
- create a db.json
- paste the data into the newly created db.json file with the following format
```
{
  "posts": [
    { "id": 1, "title": "json-server", "author": "typicode" }
  ],
  "comments": [
    { "id": 1, "body": "some comment", "postId": 1 }
  ],
  "profile": { "name": "typicode" }
}
```
- `json-server --watch db.json`

### What are the relevant tickets?
closes #61 
### Questions:

### Screenshots (if appropriate)
![Screen Shot 2021-01-11 at 1 48 59 PM](https://user-images.githubusercontent.com/56229864/104236440-c91a7c00-5413-11eb-92cf-418d40fc1c94.png)
